README.md

```diff
@@ -1,34 +1,45 @@
+<p align="center">
+    <img src="https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png" />
+    <br />
+    <br />
+    <img src="https://img.shields.io/badge/license-MIT-red.svg?style=flat" alt="MIT License" src="http://choosealicense.com/licenses/mit/" />
+    <img src="https://img.shields.io/reddit/subreddit-subscribers/manim.svg?color=orange&label=reddit" alt="Reddit" href="https://www.reddit.com/r/manim/" />
+    <img src="https://img.shields.io/discord/581738731934056449.svg?label=discord&color=yellow" alt="Discord" href="https://discord.gg/mMRrZQW" />
+    <img src="https://readthedocs.org/projects/manimce/badge/?version=latest" alt="Documentation Status" href="https://manimce.readthedocs.io/en/latest/?badge=latest" />
+    <img src="https://github.com/ManimCommunity/manim/workflows/CI/badge.svg" alt="CI" />
+    <br />
+    <br />
+    <i>An animation engine for explanatory math videos</i>
+</p>
+
+<hr />
+
+`manim` is an animation engine for explanatory math videos. It's used to create precise animations programmatically, as demonstrated in the videos of [3Blue1Brown](https://www.3blue1brown.com/).
+
+> NOTE: This repository is maintained by the Manim Community, and is not associated with Grant Sanderson or 3Blue1Brown in any way (although we are definitely indebted to him for providing his work to the world). If you would like to study how Grant makes his videos, head over to his repository ([3b1b/manim](https://github.com/3b1b/manim)). This fork is updated more frequently than that his, and it's recommended to use this fork if you'd like to use Manim for your own projects.
 
 ## Table of Contents:
 
-- [Installation](#installation)
-- [Usage](#usage)
-- [Documentation](#documentation)
-- [Help with Manim](#help-with-manim)
-- [Contributing](#contributing)
-- [License](#license)
+-  [Installation](#installation)
+-  [Usage](#usage)
+-  [Documentation](#documentation)
+-  [Help with Manim](#help-with-manim)
+-  [Contributing](#contributing)
+-  [License](#license)
 
 ## Installation
 
-Manim has a few dependencies that need to be installed before it. Please visit
-the
-[documentation](https://manimce.readthedocs.io/en/latest/installation.html)
-and follow the instructions according to your operating system.
+`manim` requires a few dependencies that must be installed prior to using it. Please visit the [documentation](https://manimce.readthedocs.io/en/latest/installation.html) and follow the appropriate instructions for your operating system.
+
+Once the dependencies have been installed, run the following in a terminal window:
```

Comment:
[optional] It may be worth changing this to "_After_ the dependencies have been installed", with the italics, because people typically expect `pip install x` to automatically handle the work of installing a python library.